source/edge-server/connect.txt

```diff
@@ -170,58 +207,155 @@ modifications you can make to the :github:`Swift template app
    .. step:: Run the App
```

Comment:
I know it's not the critical portion of this page, but if you're so inclined, the `Adjust the App Transport Security Settings (Optional)` step would benefit from having sub-steps. e.g.

> #### Adjust the App Transport Security Settings (Optional)
>   
> If your app uses TLS, you can disregard this step. If you are developing an Edge Server app with TLS not enabled during development, you must complete this step.
>  
> For iOS, Apple disables insecure connections by default. You must add an exception for your public-accessible DNS to allow the client app to connect to it:
> 
> 1. Select your App target, and go to the :guilabel:`Signing & Capabilities` tab. 
> 2. Press the :guilabel:`+ Capability` button, and search for ``App Transport Security Settings``. Add this to your app.
> 3. Add an exception domain for your public-accessible DNS. This should be just the string domain, similar to: ``ec2-13-58-70-88.us-east-2.compute.amazonaws.com``.
> ...